### PR TITLE
[SG-388] Organization menu style improvements

### PR DESCRIPTION
**Pull request by BrandonM-Bitwarden**
## Type of change
Bug fix for dark mode.

```
- [X] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

```
- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)
```

### CODE REVIEW

File: apps/web/src/scss/base.scss [hunk 1]
```diff
@@ -302,6 +302,7 @@ code {
   }
 }
 
-button i.bwi {
```
Comment (BrandonM-Bitwarden) [inline apps/web/src/scss/base.scss:305]: I would prefer if this ccs relied on `.dropdown-item` but making a change like that would require significantly more testing and may have unexpected behaviour elsewhere.

File: apps/web/src/scss/forms.scss [hunk 2]
```diff
@@ -129,7 +129,6 @@ input[type="checkbox"] {
 .dropdown-menu,
 .dropdown-item {
   @include themify($themes) {
-    background-color: themed("dropdownBackground");
```
Comment (differsthecat) [inline apps/web/src/scss/forms.scss:132]: I think the `dropdown-item` class is used in a lot of places and we probably shouldn't modify the css for it for this particular case.

**Review by differsthecat — CHANGES_REQUESTED** [review]
I'm a little wary about us modifying the css classes for things that are used in a lot of places like in `forms.scss`. This could be a good opportunity to use tailwind classes for the dropdown items here in the `organization-options` component :)

**Review by differsthecat — APPROVED** [review]
Nice work!

**Pull request merged**